modules/packages/Sort.chpl

```diff
@@ -424,7 +424,7 @@ proc heapSort(Data: [?Dom] ?eltType, comparator:?rec=defaultComparator) {
               else low + (size/2 - 1) * stride;
 
   while (start >= low) {
-    SiftDown(start, high, comparator);
+    SiftDown(high, high, comparator);
```

Comment:
I don't think you intended to change heapSort

Reply:
Yeah, it was an issue with find/replace. I just re-did my changes because I messed up and dropped the commits with my changes.

Reply:
You shouldn't need to re-implement anything - at the worst I can send you a copy of the file as you last had it, if you lost everything else